scripts/k8s_e2e_node_windows.ps1

```diff
@@ -0,0 +1,74 @@
+param (
```

Comment:
Please add `$ErrorActionPreference = "Stop"` before this.

Reply:
or after :P

Reply:
I think this was missed